inference-engine/src/inference_engine/src/cpp/ie_remote_context.cpp

```diff
@@ -59,7 +59,7 @@ std::string RemoteContext::get_device_name() const {
 
 RemoteTensor RemoteContext::create_tensor(const element::Type& element_type,
                                           const Shape& shape,
-                                          const ie::ParamMap& params) {
+                                          const ParamMap& params) {
```

Comment:
AnyMap?

Reply:
Let leave the name as is. It is still a map of parameters.  And move renaming to separate PR as the scope of this renaming is too large (up to 101 files)